lib/http/response.js

```diff
@@ -18,6 +18,7 @@ class HttpResponse extends EventEmitter {
     this.request = request;
     this.res.setEncoding('utf8');
     this.isRedirect = HttpUtil.isRedirect(response.statusCode);
+    this.isInternalServerError = response.statusCode === 500;
```

Comment:
is this a specific error case we are handling? because server errors could have status code > 500.

Reply:
For safari driver it is a specific 500 error. Goes away after a few milliseconds. 

In general I think we should retry for all 5xx errors, will update this.

Reply:
updated